vue-frontend/src/components/home/HomeUserReviewPage.vue

```diff
@@ -0,0 +1,393 @@
+<template>
+  <div class="user-reviews-div normal-text">
+    <div class="header-text canopas-gradient-text text-center">
+      Listen what real app users <br />
+      are saying...
+    </div>
+    <div class="horizontal-grid" ref="horizontalGrid">
+      <div class="grid-animation-reverse" :class="gridAnimationReverse">
+        <div
+          class="grid-animation"
+          v-on:mouseover="mouseOver"
+          v-on:mouseleave="mouseLeave"
+        >
+          <div class="horizontal-grid-1" ref="horizontalGrid1">
+            <div
+              v-for="userReview in usersReviewsGrid1"
```

Comment:
Shall we go for an alternative of not using numbers in variables? maybe userReviewUpperGrid instead of userReviewGrid1?

Reply:
I'm not sure about javascript conventions but otherwise number is fine to use in variables. For example, if we had 4 grid, number would make much more sense than word. However, if javascript does not encourage it, maybe we can rename it to `userReviewsFirstGrid`, what do you think?

Reply:
Yes, we can use numbers in words. 
It's not a javascript-specific convention I think, it's all over a recommendation we can avoid numbers if possible. Also we should avoid using digits in CSS selectors.